services/dedup/dedup_test.go

```diff
@@ -61,6 +61,7 @@ func Test_Dedup(t *testing.T) {
 func Test_Dedup_Window(t *testing.T) {
 	config.Reset()
 	logger.Reset()
+	misc.Init()
```

Comment:
[optional] We are only using `misc.CreateTMPDIR()` and `misc` is a poor package name anyway so I propose we move the function in a proper package and remove the dependency from `misc` here.

Reply:
Sounds good to me, +1 for moving it to the kit :+1:

Reply:
test keeps failing frequently, let's do the quick fix first

Reply:
Sounds good :+1: